src/directLine.ts

```diff
@@ -617,12 +626,19 @@ export class DirectLine implements IBotConnection {
             ? `${this.domain}/conversations/${this.conversationId}?watermark=${this.watermark}`
             : `${this.domain}/conversations`;
         const method = this.conversationId ? "GET" : "POST";
+        const body = this.conversationId
+            ? undefined
+            : {
+                locale: this.localeOnStartConversation
+              };
         return this.services.ajax({
             method,
             url,
+            body,
             timeout: this.timeout,
             headers: {
                 "Accept": "application/json",
+                "Content-Type": "application/json",
```

Comment:
Add a test to make sure this doesnt affect existing clients that dont send a body.